client/src/layouts/Dashboard/components/Topbar/components/SearchResults.js

```diff
@@ -54,7 +54,13 @@ const SearchResults = ({ open, results, noContent }) => {
           {results &&
             results.map((result) => (
               <>
-                <ListItem component={RouterLink} to={`/patient/${result.id}`} button key={result.id} className={classes.listItem}>
+                <ListItem
+                  component={RouterLink}
+                  to={`/patient/${result.id}`}
+                  button key={result.id}
+                  className={classes.listItem}
+                  onClick={() => !!handleClose && handleClose()}
```

Comment:
I didn't understand this condition. Can you explain this?

Reply:
It means that if handleClose prop is defined then call the function. It just a check to avoid code breakage!